config/config_test.go

```diff
@@ -7,34 +7,45 @@ import (
 	"testing"
 )
 
+const (
+	INNER                  = "inner"
+	OUTER                  = "outer"
+	INNER_VALUE            = "inner-value"
+	OUTER_INNER            = "outer.inner"
+	INNER_DEFAULT          = "inner-default"
+	INNER_DEFAULT_VALUE    = "inner-default-value"
+	PASS_THROUGH_END_POINT = "http://127.0.0.1:8080"
+	DAEMON_END_POINT       = "0.0.0.0:7000"
+)
+
 func TestCustomSubMap(t *testing.T) {
 	var config = viper.New()
-	config.Set("outer.inner", "inner-value")
-	config.SetDefault("outer.inner-default", "inner-default-value")
+	config.Set(OUTER_INNER, INNER_VALUE)
+	config.SetDefault("outer.inner-default", INNER_DEFAULT_VALUE)
```

Comment:
Should "outer.inner-default" not be changed as well?

Reply:
just used once.. so OK for now